File: src/main/java/pl/oczadly/kotlin_app/hackyeah2025/configuration/dynamo/DynamoDBProperties.java
package pl.oczadly.kotlin_app.hackyeah2025.configuration.dynamo;

import org.springframework.boot.context.properties.ConfigurationProperties;

@ConfigurationProperties(prefix = "aws.dynamodb")
public class DynamoDBProperties {

    private String region = "us-east-1";
    private String endpoint;

    public String getRegion() {
        return region;
    }

    public void setRegion(String region) {
        this.region = region;
    }

    public String getEndpoint() {
        return endpoint;
    }

    public void setEndpoint(String endpoint) {
        this.endpoint = endpoint;
    }
}

File: src/main/kotlin/pl/oczadly/kotlin_app/hackyeah2025/configuration/dynamo/DynamoDBProperties.kt
package pl.oczadly.kotlin_app.hackyeah2025.configuration.dynamo

import org.springframework.boot.context.properties.ConfigurationProperties

@ConfigurationProperties(prefix = "aws.dynamodb")
data class DynamoDBProperties(
    val region: String = "us-east-1",
    val endpoint: String? = null
)
